functions/Copy-DbaCredential.ps1

```diff
@@ -206,7 +206,7 @@ function Copy-DbaCredential {
 				$connString = "Server=ADMIN:$sourceName;Trusted_Connection=True"
 			}
 			
-			$sql = "SELECT name,credential_identity,substring(imageval,5,$ivlen) iv, substring(imageval,$($ivlen + 5),len(imageval)-$($ivlen + 4)) pass from sys.Credentials cred inner join sys.sysobjvalues obj on cred.credential_id = obj.objid where valclass=28 and valnum=2"
+			$sql = "SELECT REPLACE(name,']',']]'),credential_identity,substring(imageval,5,$ivlen) iv, substring(imageval,$($ivlen + 5),len(imageval)-$($ivlen + 4)) pass from sys.Credentials cred inner join sys.sysobjvalues obj on cred.credential_id = obj.objid where valclass=28 and valnum=2"
```

Comment:
Wouldn't just wrapping name in `QUOTENAME()` be sufficient?

Reply:
It appears as though it would be yes. I'll modify the code.